Visual Studio Project Template C#/PluginInfrastructure/DllExport/NppPlugin.DllExport.targets

```
@@ -30,15 +30,17 @@
 		On 32 bit windows usually C:\Program Files\
 		On 64 bit windows usually C:\Program Files (x86)\
 		$(ProgramW6432) points to the 64bit Program Files (on 32 bit windows it is blank) -->
+    <MakeDir Directories="$(MSBuildProgramFiles32)\Notepad++\plugins\$(TargetName)\" Condition="Exists('$(MSBuildProgramFiles32)\Notepad++\plugins\') AND !Exists('$(MSBuildProgramFiles32)\Notepad++\plugins\$(TargetName)\') AND '$(Platform)'=='x86'" />
     <Copy 
         SourceFiles="$(TargetPath)" 
-        DestinationFolder="$(MSBuildProgramFiles32)\Notepad++\plugins\" 
-        Condition="Exists('$(MSBuildProgramFiles32)\Notepad++\plugins\') AND '$(Platform)'=='x86'"
-        ContinueOnError="false" /> 
+        DestinationFolder="$(MSBuildProgramFiles32)\Notepad++\plugins\$(TargetName)\)" 
```

After this change it works like a charm!
```suggestion
        DestinationFolder="$(MSBuildProgramFiles32)\Notepad++\plugins\$(TargetName)\" 
```

---

Ho yes.

Bad copy paste I presume.